src/asynk/async_queue.rs

```diff
@@ -159,10 +163,16 @@ where
     <Tls as MakeTlsConnect<Socket>>::TlsConnect: Send,
     <<Tls as MakeTlsConnect<Socket>>::TlsConnect as TlsConnect<Socket>>::Future: Send,
 {
+    #[builder(default=None ,  setter(skip))]
+    pool: Option<Pool<PostgresConnectionManager<Tls>>>,
     #[builder(setter(into))]
-    pool: Pool<PostgresConnectionManager<Tls>>,
+    uri: String,
+    #[builder(setter(into))]
+    number_conns: u32,
```

Comment:
maybe `max_pool_size`?

Reply:
Solve in [20e4757](https://github.com/ayrat555/fang/pull/54/commits/20e4757e6412d6819109d60b1ab308ded0863332)